code/game/machinery/iv_drip.dm

```diff
@@ -39,13 +39,17 @@
 			. += filling
 
 /obj/machinery/iv_drip/MouseDrop(mob/living/target)
-	if(usr.incapacitated())
+	var/mob/user = usr
+	if(HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
 		return
 
-	if(!ishuman(usr) || !iscarbon(target))
+	if(!ishuman(user) || !iscarbon(target))
```

Comment:
```suggestion
	if(!ishuman(user))
```
don't bother checking to see if they're carbon here, it's already implied if they fail the ishuman check